sandbox.c

```diff
@@ -30,6 +30,32 @@ static void add_syscall(scmp_filter_ctx ctx, const char *syscall, uint32_t actio
 	}
 }
 
+static const char *get_seccomp_action_name(uint32_t action)
+{
+        const char *action_name;
+        switch (action) {
+            case SCMP_ACT_KILL_PROCESS:
```

Comment:
currently we only support `SCMP_ACT_KILL_PROCESS`, `SCMP_ACT_LOG` and `SCMP_ACT_ALLOW`

Let's drop others for now to avoid confusion for people looking into the source code. They probably should be added as we onboard new action types.

Reply:
Yep, makes sense.